.github/workflows/v1.yml

```diff
@@ -0,0 +1,55 @@
+name: Run Tests
```

Comment:
(Non blocking and maybe something for the future)

Can we parameterize v1 and v2 yml somehow to reuse the  logic in there? The diff seems not very rich:
```diff
6c6
<       v1
---
>       master
48,49c48,49
<           go run build.go gfmrun docs/v1/manual.md
<           go run build.go toc docs/v1/manual.md
---
>           go run build.go gfmrun docs/v2/manual.md
>           go run build.go toc docs/v2/manual.md
```

Reply:
Done